docs/runtime_CLI.md

```diff
@@ -317,6 +320,97 @@ the meter type is packets, the rate is entered in packets/microsecond and
 burst_size is the number of packets.
 
 
+### mirroring_add, mirroring_add_mc, mirroring_delete, mirroring_get
+
+[simple_switch_CLI only]
+
+When any of these events happened during packet processing:
+
++ Your P4_16 v1model architecture program called `clone` or `clone3`
+  extern functions during ingress or egress processing, and now
+  ingress or egress processing has completed
++ Your P4_14 program called `clone_ingress_pkt_to_egress` during
+  ingress processing, which is now complete
++ Your P4_14 program called `clone_egress_pkt_to_egress` during egress
+  processing, which is now complete
+
+then not only will the current "original" packet continue with
+whatever it normally does, as if your program had not called a clone
+operation, but the PRE (Packet Replication Engine) part of
+`simple_switch` will also create zero or more copies, or clones, of
+the current packet, and cause them to be enqueued in the packet buffer
+such that later each of the cloned packet(s) will perform egress
+processing.  Each of the clones is processed independently from each
+other, and independently from the original packet.
+
+Search for occurrences of the word "clone" in [this
+article](https://github.com/p4lang/behavioral-model/blob/master/docs/simple_switch.md#pseudocode-for-what-happens-at-the-end-of-ingress-and-egress-processing)
```

Comment:
please just use `(/docs/simple_switch.md#pseudocode-for-what-happens-at-the-end-of-ingress-and-egress-processing)` as the link

Reply:
I tried without the `/docs/` and it works when clicking the links, and I think is the shortest path that works.  Let me know if you really want the `/docs/` in the two links that I changed in the last two commits.

Reply:
Without docs/ is good too